django_netjsongraph/__init__.py

```diff
@@ -1,4 +1,4 @@
-VERSION = (0, 6, 3, 'final')
+VERSION = (0, 6, 3, 'b')
```

Comment:
Sorry ajay, I gave a wron suggestion, it shoudn't be 0.6.3b but 0.6.3 post1:
https://www.python.org/dev/peps/pep-0440/#post-releases